lib/activerecord-import/import.rb

```diff
@@ -275,7 +275,11 @@ def import_helper( *args )
           # this next line breaks sqlite.so with a segmentation fault
           # if model.new_record? || options[:on_duplicate_key_update]
             column_names.map do |name|
-              model.read_attribute_before_type_cast(name.to_s)
+              if model.class.column_defaults[name.to_s].is_a? Integer
+                model.read_attribute(name.to_s)
+              else
+                model.read_attribute_before_type_cast(name.to_s)
```

Comment:
The original commit was back in early 2007: https://github.com/zdennis/ar-extensions/commit/b993d23533e8342d856377fd77006127679ba569

The commit message kind of sucks so I apologize for that. 

IIRC, it's because we are collecting DB-safe representations of the values and ActiveRecord may give you back an object that's nice to work with in Ruby but that may not be db-safe. I think Date, DateTime, and boolean fields were an example of this.

But, if we can remove it and have the test suite continue to pass with just `read_attribute` I'm all for it.